### ISSUE TITLE
你好，方便提供一下可视化的代码吗

### ISSUE THREAD

**Issue by roogko** [RaccoonDML/CHSNet#2]
密度图可视化

**Comment by RaccoonDML** [comment]
灰度图用colormap表示颜色
`
import numpy as np
from PIL import Image
import matplotlib.pyplot as plt
import torch

def tensor2image(x, color_map='jet'):
    """
    tensor: numpy.float or tensor in shape of (H, W)
    """
    if isinstance(x, torch.Tensor):
        x = x.numpy()
    x = x/x.max() # normalize to 0~1
    cmap = plt.get_cmap(color_map)
    x = (cmap(x) * 255).astype(np.uint8)
    output = Image.fromarray(x)
    return output
`

**Comment by roogko** [comment]
> 灰度图用colormap表示颜色 ` import numpy as np from PIL import Image import matplotlib.pyplot as plt import torch
> 
> def tensor2image(x, color_map='jet'): """ tensor: numpy.float or tensor in shape of (H, W) """ if isinstance(x, torch.Tensor): x = x.numpy() x = x/x.max() # normalize to 0~1 cmap = plt.get_cmap(color_map) x = (cmap(x) * 255).astype(np.uint8) output = Image.fromarray(x) return output `

感谢！
